fast64_internal/oot/oot_actor.py

```diff
@@ -360,15 +368,17 @@ def drawDetailedProperties(user, userProp, userLayout, userObj, userSearchOp, us
 	else:
 		# If the current actor is custom
 		if user != userEntrance:
-			prop_split(userLayout, detailedProp, userIDField + 'Custom', currentActor)
-			prop_split(userLayout, detailedProp, userParamField + 'Custom', 'Actor Parameter')
-			userLayout.prop(detailedProp, 'rotOverrideCustom', text = 'Override Rotation (ignore Blender rot)')
-			if detailedProp.rotOverrideCustom:
-				if user == userActor: prop_split(userLayout, detailedProp, 'rotOverrideXCustom', 'Rot X')
-				prop_split(userLayout, detailedProp, 'rotOverrideYCustom', 'Rot Y')
-				if user == userActor: prop_split(userLayout, detailedProp, 'rotOverrideZCustom', 'Rot Z')
+			prop_split(userLayout, detailedProp, getCustomPropName(userIDField), currentActor)
+			prop_split(userLayout, detailedProp, userParamField, 'Actor Parameter')
```

Comment:
```suggestion
			prop_split(userLayout, detailedProp, getCustomPropName(userParamField), 'Actor Parameter')
```
?
but also I don't think you need `getCustomPropName` anywhere actually, using the non-custom prop with a custom actor will use the custom prop thanks to getValues/setValues

Reply:
yeah I'm already doing that (the get/set thing), that's why I didn't use ``getCustomPropName``

Reply:
it's needed for `userIDField` and in other spots though?

Reply:
yeah because it's (blender) properties that doesn't use get/set at all as it's not needed

Reply:
so why do these have a Custom variant at all then?